googletest/include/gtest/internal/gtest-port.h

```diff
@@ -2505,6 +2506,7 @@ typedef TypeWithSize<4>::Int Int32;
 typedef TypeWithSize<4>::UInt UInt32;
 typedef TypeWithSize<8>::Int Int64;
 typedef TypeWithSize<8>::UInt UInt64;
+typedef TypeWithSize<sizeof(void*)>::UInt UIntPtr;
```

Comment:
You're right. I should look more closely before commenting.
`uintptr_t` is an "optional" definition in <cstdint>.
I guess we should stick to what you've defined it as.
Should you also define a signed `IntPtr`?

We really need to start killing off obsolete compiler support!